site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
 ---
 layout: "layouts/doc-post.njk"
-title: Set up Group Publishing
+title: Set up group publishing
```

Comment:
Is it more like a group ownership than group publishing? 

To me this is more like: Adding more developers to your publisher account. And to do this, you need to create a group publisher first. 

On the other hand, "group publishing" sounds like you do a group of publishing (as in batches).

Reply:
What about "Set up a group publisher" or even "Set up a publishing group"?

Reply:
I like the first option better. Thanks!

Reply:
Done.